app/controllers/hyku/api/v1/work_controller.rb

```diff
@@ -74,6 +75,12 @@ def no_result_message
             # default message
             "This tenant has no works"
           end
+
+          def work_presenter_class(doc)
+            model_name = doc.to_model.model_name.name
+            return Hyku::WorkShowPresenter if model_name == "GenericWork"
+            "Hyrax::#{model_name}Presenter".safe_constantize
```

Comment:
How about having `Hyku::WorkShowPresenter` as a fallback?
```suggestion
            "Hyrax::#{model_name}Presenter".safe_constantize || Hyku::WorkShowPresenter
```